templates/appbase.html

```diff
@@ -1,53 +0,0 @@
-<!doctype html>
-<html lang="en">
-<head>
-    <!-- Required meta tags -->
-    <meta charset="utf-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
-
-    <!-- Bootstrap CSS -->
-    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css" integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
-    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap-select@1.13.7/dist/css/bootstrap-select.min.css">
-    <title> {% block title %}App base{% endblock %}</title>
-
-    {% block style %}{% endblock %}
-    {% block head %}{% endblock head %}
```

Comment:
I had added this here so that a template can extend the <head> of the base (was also used in `histogram_file_manager`), so we should either include ALL templates' extra JS/CSS in the base (not really necessary), or restore the blocks

Reply:
Same for `block style` above!

Reply:
I think we should restore the blocks. They make perfect sense and I didn't realize they were in appbase and not base. The goal being to have only one of the two. Sorry about that (again :| ).

Reply:
I totally agree with just a `base.html`, much cleaner